### PR TITLE
75280: Improve Helm Deployment Content

### PR DESCRIPTION
**Pull request by jonquil2002**
SC: https://app.shortcut.com/replicated/story/75280/update-the-about-deploying-with-helm-topic

Preview: https://deploy-preview-1114--replicated-docs-upgrade.netlify.app/vendor/helm-overview

### CODE REVIEW

File: docs/vendor/helm-overview.md [hunk 1]
```diff
@@ -132,7 +170,7 @@ The resulting deployment is comprised of standard Kubernetes manifests. Therefor
 The app manager supports native Helm and Replicated Helm installations into air gap environments. When a user installs a Helm chart-based application in an air gap environment, the chart processing is managed in the end user environment. This means that the app manager can use user-supplied values, license values, and existing values to create deployable manifests. For more information, see [`builder`](/reference/custom-resource-helmchart#builder) in the _HelmChart_ reference.
 
 
-## helm CLI (Beta)
+## helm CLI (Beta) Deployment
```
Comment (paigecalvert) [inline docs/vendor/helm-overview.md:173]: helm CLI **(Beta)** Deployment > helm CLI Deployment **(Beta)**

**Review by paigecalvert — COMMENTED** [review]
@jonquil2002 some questions and small nits!

File: docs/vendor/helm-overview.md [hunk 2]
```diff
@@ -15,7 +15,45 @@ Helm is a popular package manager for Kubernetes applications. Using Replicated
...
+  <th width="15%">Installation Type</th>
+  <th width="15%">Uses App Manager?</th>
+  <th width="15%">Supported On</th>
+  <th width="55">Highlights</th>
+</tr>
+<tr>
+  <td>Native Helm</td>
+  <td>Yes</td>
+  <td><ul><li>Existing clusters</li><li>Kubernetes installer clusters</li><li>Air gap</li></ul></td>
+  <td><ul><li>Recommended type for app manager</li><li>Supports more Helm options, including hooks and weights</li><li>App manager manages the lifecycle of the application</li></ul></td>
```
[the remark this answers is not in the document]
Comment (jonquil2002) [reply docs/vendor/helm-overview.md:33]: Maybe "Installs with App Manager?"

File: docs/vendor/helm-overview.md [hunk 3]
```diff
@@ -15,7 +15,45 @@ Helm is a popular package manager for Kubernetes applications. Using Replicated
...
+  <td><ul><li>Limited Replicated features</li><li>Helm manages the lifecycle of the application</li></ul></td>
+</tr>
+</table>
+
+## Limitations
+
+There are different limitations depending on if your customers install and manage the application with the app manager or if they use the helm CLI directly. For more information, see:
+
+* [Supporting Native Helm and Replicated Helm](helm-release)
+* [Supporting helm CLI Installations (Beta)](helm-install)
```
[the remark this answers is not in the document]
Comment (jonquil2002) [reply docs/vendor/helm-overview.md:54]: Yes, the limitations are so long and make the goal of this topic rather convoluted. It also seems redundant to have them in two places. They seem more relevant in the "Supporting" topics.

Changed the links back to linking to those sections specifically.

**Review by ajp-io — APPROVED** [review]

**Review by paigecalvert — APPROVED** [review]
lgtm!

**Pull request merged**